### PR TITLE
Compile less

### PR DESCRIPTION
**Pull request by pgiraud**
Using less files in the browser is a pain in the ass. With the current pull request my goal is to build the css files each time a less file is changed when using `make serve`.

I'm not completely sure the code is pretty and would appreciate a review especially the `Makefile` part.

For example, I wonder why the following statements appear in the console:
```
make[1]: Entering directory `/home/pierre/public_html/ngeo'
make[1]: Leaving directory `/home/pierre/public_html/ngeo'
```

### CODE REVIEW

**Comment by sbrunner** [comment]
This pullrequest looks to completely breaks the integration with c2cgeoportal ...

**Comment by pgiraud** [comment]
Can you elaborate?

**Comment by sbrunner** [comment]
You delete the build.less who is required ... to build the mobile

File: Makefile [hunk 1]
```diff
@@ -95,7 +94,7 @@ check: lint dist check-examples test compile-examples build-gmf-mobile-app
 compile-examples: .build/examples/all.min.js
 
 .PHONY: build-gmf-mobile-app
-build-gmf-mobile-app: $(addprefix contribs/gmf/build/mobile,.js .css)
+build-gmf-mobile-app: contribs/gmf/build/mobile.js compile-css
```
Comment (sbrunner) [inline Makefile]: This change is wrong when we will add the desktop app

**Comment by pgiraud** [comment]
`build.less` is actually replaced by `mobile.less`.

**Comment by sbrunner** [comment]
The build.less will include a file that is modifiable by the project and mobile.less is the generic part than they can't be merge!

**Comment by pgiraud** [comment]
There are 2 different `mobile.less` files.
`contribs/gmf/apps/mobile/less/mobile.less` and `contribs/gmf/less/mobile.less`.

Instead of keeping `build.less` and `mobile.less` in the apps/mobile directory, I propose we keep only one. The `mobile.less` file in `apps/mobile` would be the customizable one.

**Comment by pgiraud** [comment]
@sbrunner Please review again.

**Comment by sbrunner** [comment]
Looks good for me :-)

**Pull request merged**